content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Linux पर bash ऑटो-कम्पलीशन"
+description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+## परिचय
+
+Bash के लिए kubectl  कम्पलीशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्पलीशन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्पलीशन ऐनेबल हो जाती है।
+
+हालाँकि, कम्पलीशन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर करती हैं जिसका अर्थ है कि आपको पहले इस सॉफ़्टवेयर को स्थापित करना होगा (आप `type _init_completion`  चलाकर परीक्षण कर सकते हैं कि आपने  पहले से bash-completion इंस्टॉल की है या नहीं)।
```

Comment:
> हालाँकि, कम्पलीशन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर करती हैं

Can be rephrased for grammatical correctness as below

हालाँकि, कम्पलीशन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर हैं